src/components/panel/panel.scss

```diff
@@ -22,3 +18,51 @@
   position: fixed;
   top: 0;
 }
+
+// Only used when no animations are present.
+._md-panel-shown .md-panel {
+  opacity: 1;
+  transition: none;
+}
+
+.md-panel {
+  opacity: 0;
+  overflow: auto;
+
+  &._md-panel-shown {
+    // Only used when custom animations are present.
+    // Overridden by the default animations.
+    opacity: 1;
+    transition: none;
+
+    &._md-panel-animate-slide-in {
+      opacity: 1;
+      transition: $swift-ease-out, opacity 1ms;
```

Comment:
Isn't the $swift-ease-out the material design ones? They are the ones used in dialog.